### PR TITLE
Update zookeeper client

### PR DESCRIPTION
**Pull request by 1u0**
# One-line summary

## Description
Update `org.apache.zookeeper` dependency to `3.6.1` (to reduce noisy logging) which includes fix for:
https://issues.apache.org/jira/browse/ZOOKEEPER-4376

## Deployment Notes
This change should be compatible with the Zookeeper server `3.6.1`.

### CODE REVIEW

File: api-metastore/src/main/java/org/zalando/nakadi/model/AuthorizationAttributeQueryParser.java [hunk 1]
```diff
@@ -1,6 +1,6 @@
 package org.zalando.nakadi.model;
 
-import org.apache.commons.lang.StringUtils;
+import org.apache.commons.lang3.StringUtils;
```
Comment (a1exsh) [inline api-metastore/src/main/java/org/zalando/nakadi/model/AuthorizationAttributeQueryParser.java:3]: Is it a compile- or a run-time error if we don't change this?

Comment (a1exsh) [reply api-metastore/src/main/java/org/zalando/nakadi/model/AuthorizationAttributeQueryParser.java:3]: I notice we are already using a mix of the two (in different sub-projects):
```
import org.apache.commons.lang.StringUtils;
import org.apache.commons.lang3.StringUtils;
```

That's not good.  We should fix it, but I'd prefer to do that separately from bumping ZooKeeper version.

Comment (a1exsh) [reply api-metastore/src/main/java/org/zalando/nakadi/model/AuthorizationAttributeQueryParser.java:3]: Ah, I see now that these are the only two uses of `lang` that are left there. OK.

**Comment by a1exsh** [comment]
:+1:

File: api-metastore/src/main/java/org/zalando/nakadi/model/AuthorizationAttributeQueryParser.java (hunk 1 above)
Comment (1u0) [reply api-metastore/src/main/java/org/zalando/nakadi/model/AuthorizationAttributeQueryParser.java:3]: That would be compile error, so the first commit is actually per-requisite for the version bump.
The updated Zookeeper client version is not using the old `org.apache.commons` lib anymore. The Nakadi project was using it as transitive dependency.

**Comment by samizzy** [comment]
:+1:

**Comment by rishiraj88** [comment]
Good PR.

**Pull request merged**